mkosi.md

```diff
@@ -272,73 +272,59 @@ prefixed with `!`, if any later assignment of that setting tries to add
 the same value, that value is ignored. Values prefixed with `!` can be
 globs to ignore more than one value.
 
-To conditionally include configuration files, the `[Match]` section can
-be used. A configuration file is only included if all the conditions in the
-`[Match]` block are satisfied. If a condition in `[Match]` depends on a
-setting and the setting has not been explicitly configured when the condition
-is evaluated, the setting is assigned its default value.
-
-Command line options that take no argument are shown without "=" in
-their long version. In the config files, they should be specified with
-a boolean argument: either "1", "yes", or "true" to enable, or "0",
-"no", "false" to disable.
+To conditionally include configuration files, the `[Match]` section can be used. If multiple matches are
+specified, the config will be included if all of them apply (i.e. a logical AND is applied). Matches can use
+a pipe symbol ("|") after the equals sign ("…=|…"), which causes the match to become a triggering match. If
+at least one triggering match is defined for a config file, then the config file will be included if at least
+one of the triggering matches of the config file applies and all of the regular (i.e. non-triggering) matches
+apply. To negate the result of a match, prefix the argument with an exclamation mark. If you prefix an
```

Comment:
Let's drive this home harder by writing something along the lines that "a `[Match]` secion is included if the logical AND of all non-triggering AND the logical OR of all triggering matches is satisfied".